### ISSUE TITLE
Use FSA standard

### ISSUE THREAD

**Issue by leonardodino** [colingourlay/react-local-store#1]
Hey there :wave:
This is a really nice approach!

I've done something [similar](https://github.com/leonardodino/context-storage), albeit simpler and akin to `useState` a couple of months ago.

Nice to see `sync` already is implemented (:

---

I know it's already `v1` and such...
But is there a reason not to adhere to [FSA](https://github.com/redux-utilities/flux-standard-action)?

It's nice to reuse available `redux` tooling and ecosystem.
I know `payload` is almost double the character size of `data`,
but there are [ways around writing it](https://github.com/redux-utilities/redux-actions) in your app code.

**Comment by colingourlay** [comment]
Hey!

There was no reason for me to not use FSA, other than me not knowing about it!

Thanks for bringing it to my attention. I'm all for making things more interoperable, so I've just updated the interface and made a v2. It's early days in this project's life, so totally not an issue to change it at this stage.

Thanks for taking the time to report this. I really appreciate it!

Colin

P.S. I like your approach to this problem too. It's a great alternative for those not wanting to use reducers. I've linked it in my README.
